src/test/java/bio/terra/workspace/common/BaseConnectedTest.java

```diff
@@ -11,4 +11,7 @@
  */
 @Tag("connected")
 @ActiveProfiles("connected-test")
-public class BaseConnectedTest extends BaseTest {}
+public class BaseConnectedTest extends BaseTest {
+
+  public static final String bufferServiceDisabledEnvsRegEx = "alpha|staging|prod";
```

Comment:
iiuc we'll remove this env-based disabling as we deploy buffer service to other environments. 
I realize it's a lot of overhead, but for the sake of this not getting lost could you create a ticket for removing this and add a todo here pointing to that ticket?

Reply:
Is it alright to put that in the PF board? Since you folks will be the ones working on buffer service?

Reply:
sgtm, just made PF-507 for this

Reply:
Thank you!